Source/WebInspectorUI/UserInterface/Controllers/TimelineManager.js

```diff
@@ -830,28 +830,28 @@ WI.TimelineManager = class TimelineManager extends WI.Object
             console.assert(isNaN(endTime));
 
             // Pass the startTime as the endTime since this record type has no duration.
-            return new WI.LayoutTimelineRecord(WI.LayoutTimelineRecord.EventType.InvalidateStyles, startTime, startTime, callFrames, sourceCodeLocation);
+            return new WI.LayoutTimelineRecord(WI.LayoutTimelineRecord.EventType.InvalidateStyles, startTime, startTime, stackTrace, sourceCodeLocation);
```

Comment:
Do we need to do something to version timeline recording exports?

Reply:
ideally yes, but AFAICT we dont include `callFrames` right now anyways (see `WI.LayoutTimelineRecord.prototype.toJSON` and `WI.ScriptTimelineRecord.prototype.toJSON`), so there's not really a compatibility problem